fullcontact/__about__.py

```diff
@@ -0,0 +1,16 @@
+__all__ = [
+    "__name__",
+    "__version__",
+    "__author__",
+    "__author_email__",
+    "__description__",
+    "__license__"
+]
+
+__name__ = "python-fullcontact"
```

Comment:
`fullcontact-python` seems better. And use the same patter in all other client libraries.

Reply:
Thanks, HJ! I'll change that. @prakhar-j , can we follow this naming convention in the java library as well?

Reply:
Assuming this is for publishing, I have name as `'Client libraries for FullContact v3 APIs'`

Reply:
I have changed the name to `fullcontact-python`, since this version is not backward compatible anyway.

Reply:
I had to revert this change, as a package with name `fullcontact-python` already exists in Pypy: https://pypi.org/project/fullcontact-python/